translate/rus.json

```diff
@@ -242,13 +242,13 @@
 "moob volume":"громкость муба",
 "moob":"муб",
 "more":"больше",
-"move height copy":"высота ходов копия",
-"move height live":"высота ходов жив",
-"move height pv":"высота ходов ов",
-"move height pva":"высота ходов ова",
+"move height copy":"высота копии ходов",
+"move height live":"высота окошек ов",
+"move height pv":"высота окошек ов",
+"move height pva":"высота окошка ова",
 "move height":"высота ходов",
-"moves copy":"копии ходов",
-"moves live":"ходы хив",
+"moves copy":"копия ходов",
+"moves live":"показать ов движков",
```

Comment:
Line 250: We don't have multiple copies of moves appearing, right? That is why I changed "копия" noun from plural to singular form.
Line 251: As from my point of view, disabling "moves live" just disables engines' PV off from showing to our screen. That is why I translated this line as "show engines' PVs" You might consider changing the line in main Eng Loc as well, but I guess this would require new translations for all the other languages as well, so think wisely.